code/datums/emotes.dm

```diff
@@ -0,0 +1,353 @@
+	message_untargeted = "#SOURCE giggles."
+	display_flags = EMOTE_AUDIBLE | EMOTE_VISIBLE | EMOTE_DISALLOW_VOICELESS
+
+/datum/emote/point
+	id = "point"
+	message_targeted = "#SOURCE points at #TARGET."
+	message_untargeted = "#SOURCE points in front of them."
+	display_flags = EMOTE_VISIBLE | EMOTE_DISALLOW_HANDLESS
+
+/datum/emote/beep
+	id = "beep"
+	message_targeted = "#SOURCE beeps at #TARGET."
+	message_untargeted = "#SOURCE beeps."
+	display_flags = EMOTE_AUDIBLE | EMOTE_DISALLOW_ORGANIC
+
+/datum/emote/scream
+	id = "scream"
+	message_targeted = "#SOURCE screams at #TARGET."
+	message_untargeted = "#SOURCE screams!"
+	display_flags = EMOTE_AUDIBLE | EMOTE_VISIBLE | EMOTE_DISALLOW_VOICELESS
+
+/datum/emote/collapse
+	id = "collapse"
+	message_targeted = "#SOURCE collapses at the sight of #TARGET!"
+	message_untargeted = "#SOURCE collapses!"
+	display_flags = EMOTE_AUDIBLE | EMOTE_VISIBLE | EMOTE_DISALLOW_VOICELESS
+
+/datum/emote/collapse/do_emote(var/atom/source, var/atom/target, var/untargeted = message_untargeted, var/targeted = message_targeted)
+	//. = ..()
+	//var/mob/M = source
+	//M.Weaken(5)
+
+/datum/emote/whisper
+	id = "whisper"
+	message_targeted = "#SOURCE whispers towards #TARGET."
+	message_untargeted = "#SOURCE whispers."
+	display_flags = EMOTE_AUDIBLE | EMOTE_VISIBLE | EMOTE_DISALLOW_VOICELESS
+
+/datum/emote/twitch
+	id = "twitch"
+	message_targeted = "#SOURCE flinches."
+	message_untargeted = "#SOURCE twitches."
+	display_flags = EMOTE_AUDIBLE | EMOTE_VISIBLE
+
+/datum/emote/gasp
+	id = "gasp"
+	message_targeted = "#SOURCE gasps at #TARGET."
+	message_untargeted = "#SOURCE gasps."
+	display_flags = EMOTE_AUDIBLE | EMOTE_VISIBLE
+
+/datum/emote/gasp_heavy
+	id = "gasp_heavy"
+	message_targeted = "#SOURCE gasps for air!"
+	message_untargeted = "#SOURCE gasps for air!"
+	display_flags = EMOTE_AUDIBLE | EMOTE_VISIBLE
+
+/datum/emote/yawn
+	id = "yawn"
+	message_targeted = "#SOURCE yawns at #TARGET."
+	message_untargeted = "#SOURCE yawns."
+	display_flags = EMOTE_AUDIBLE | EMOTE_VISIBLE
+
+/datum/emote/snore
+	id = "snore"
+	message_targeted = "#SOURCE sarcasticly snores at #TARGET."
+	message_untargeted = "#SOURCE snores."
+	display_flags = EMOTE_AUDIBLE | EMOTE_VISIBLE
+
+/datum/emote/doze
+	id = "doze"
+	message_targeted = "#SOURCE has trouble keeping their eyes open."
+	message_untargeted = "#SOURCE has trouble keeping their eyes open."
+	display_flags = EMOTE_VISIBLE | EMOTE_DISALLOW_MACHINE
+
+/datum/emote/limp
+	id = "limp"
+	message_targeted = "#SOURCE's body becomes limp."
+	message_untargeted = "#SOURCE's body becomes limp."
+	display_flags = EMOTE_VISIBLE
+
+/datum/emote/bounce
+	id = "bounce"
+	message_targeted = "#SOURCE bounces towards #TARGET."
+	message_untargeted = "#SOURCE bounces in place."
+	display_flags = EMOTE_VISIBLE
+
+/datum/emote/sway
+	id = "sway"
+	message_targeted = "#SOURCE sways towards #TARGET."
+	message_untargeted = "#SOURCE sways."
+	display_flags = EMOTE_VISIBLE
+
+/datum/emote/light
```

Comment:
Has this emote always existed? It sounds like a pretty poor addition. Adding custom emotes tends to be pretty questionable in all cases, but this one seems both clumsy and niche.